contracts/LinniaRecords.sol

```diff
@@ -148,6 +154,57 @@ contract LinniaRecords is Ownable, Pausable, Destructible {
         return true;
     }
 
+    /* @dev allows a record owner to add a policy to a record as long as it conforms */
+    /* @param dataHash the record the owner is adding a policy to */
+    /* @param policy an address of a contract that conforms to the policy interface to add */
+    function addPolicyToRecord(bytes32 dataHash, address policy)
+        whenNotPaused
+        onlyUser
+        onlyRecordOwnerOf(dataHash)
+        public
+        returns (bool)
+    {
+
+        /* @dev original record must be policy complaint to work */
+        require(hub.policiesContract().policyIsValid(
+            dataHash,
+            msg.sender,
+            records[dataHash].dataUri,
+            policy
+        ));
+
+        policies[dataHash].push(policy);
```

Comment:
A few things:
1. It would be good to revert if the policy has already been added.
2. Also would be good to check for address(0)
3. It would be good to check that this address conforms to the interface (not sure how todo this)